packer/telemetry.go

```diff
@@ -116,20 +119,23 @@ func (c *CheckpointTelemetry) Finalize(command string, errCode int, err error) e
 		extra.Error = err.Error()
 	}
 	params.Payload = extra
+	// b, _ := json.MarshalIndent(params, "", "    ")
+	// log.Println(string(b))
```

Comment:
Need these commented lines?

Reply:
it's a good way of debugging and sufficiently difficult to reproduce that I think we should just leave it in